recipes/sdl2_mixer/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Make sure TravisCI can find SDL2
+if [ `uname` == Darwin ]; then
+	export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib"
```

Comment:
I don't know how to do this for a generic approach (e.g. in which file should I place this code). Wouldn't it be beter if I'd post an issue instead?